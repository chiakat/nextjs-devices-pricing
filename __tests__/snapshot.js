// __tests__/snapshot.js
import React from 'react';
import renderer from 'react-test-renderer';
import Index from '../pages/index';
import { SessionProvider } from 'next-auth/react';


it('renders homepage unchanged', () => {
  const tree = renderer.create(<SessionProvider><Index /></SessionProvider>).toJSON();
  expect(tree).toMatchSnapshot();
});
