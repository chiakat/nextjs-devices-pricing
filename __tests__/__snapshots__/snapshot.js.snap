// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders homepage unchanged 1`] = `
<div>
  <header
    className="MuiPaper-root MuiPaper-elevation MuiPaper-elevation0 MuiAppBar-root MuiAppBar-colorPrimary MuiAppBar-positionStatic css-11lf942-MuiPaper-root-MuiAppBar-root"
  >
    <div
      className="MuiToolbar-root MuiToolbar-gutters MuiToolbar-regular css-1t29gy6-MuiToolbar-root"
    >
      <img
        alt="alwaysAI Logo"
        className="MuiBox-root css-vcjgua"
        src="/images/aai-logo-black.svg"
      />
      <div
        className="MuiBox-root css-otv6bo"
      />
      <div
        className="MuiTypography-root MuiTypography-h6 css-1juivf6-MuiTypography-root"
      >
        <a
          href="/"
          onClick={[Function]}
          onMouseEnter={[Function]}
        >
          Products
        </a>
      </div>
      <div
        className="MuiTypography-root MuiTypography-h6 css-1juivf6-MuiTypography-root"
      >
        <a
          href="/"
          onClick={[Function]}
          onMouseEnter={[Function]}
        >
          Use Cases
        </a>
      </div>
      <div
        className="MuiTypography-root MuiTypography-h6 css-1juivf6-MuiTypography-root"
      >
        <a
          href="/pricing"
          onClick={[Function]}
          onMouseEnter={[Function]}
        >
          Pricing
        </a>
      </div>
      <div
        className="MuiTypography-root MuiTypography-h6 css-1juivf6-MuiTypography-root"
      >
        <a
          href="/"
          onClick={[Function]}
          onMouseEnter={[Function]}
        >
          Blog
        </a>
      </div>
      <div
        className="MuiTypography-root MuiTypography-h6 css-1juivf6-MuiTypography-root"
      >
        <a
          href="/"
          onClick={[Function]}
          onMouseEnter={[Function]}
        >
          Support
        </a>
      </div>
      <div
        className="MuiBox-root css-1wveqnu"
      />
      <div
        className="MuiTypography-root MuiTypography-h6 css-1juivf6-MuiTypography-root"
      >
        <a
          href="/devices/view"
          onClick={[Function]}
          onMouseEnter={[Function]}
        >
          Devices
        </a>
      </div>
      <div
        className="MuiTypography-root MuiTypography-h6 css-1juivf6-MuiTypography-root"
      >
        <a
          href="/"
          onClick={[Function]}
          onMouseEnter={[Function]}
        >
          Models
        </a>
      </div>
      <div
        className="MuiTypography-root MuiTypography-h6 css-1juivf6-MuiTypography-root"
      >
        <a
          href="/"
          onClick={[Function]}
          onMouseEnter={[Function]}
        >
          Docs
        </a>
      </div>
      <div
        className="MuiTypography-root MuiTypography-h6 css-1juivf6-MuiTypography-root"
      >
        <a
          href="/api/auth/signin"
          onClick={[Function]}
          onMouseEnter={[Function]}
        >
          Log in
        </a>
      </div>
    </div>
  </header>
  <main>
    <div
      className="MuiContainer-root MuiContainer-maxWidthLg css-12n9p2n-MuiContainer-root"
    >
       
       
    </div>
  </main>
  <footer
    className="MuiTypography-root MuiTypography-body1 css-a9m5nw-MuiTypography-root"
  >
    <div
      className="MuiContainer-root MuiContainer-maxWidthLg css-1wu66hi-MuiContainer-root"
    >
      <div
        className="MuiGrid-root MuiGrid-container MuiGrid-spacing-xs-5 css-1ld3b9g-MuiGrid-root"
      >
        <div
          className="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-6 MuiGrid-grid-sm-4 MuiGrid-grid-md-3 css-180mmmm-MuiGrid-root"
        >
          <img
            alt="alwaysAI Logo"
            className="MuiBox-root css-nicvk6"
            src="/images/aai-logo-white.svg"
          />
          <p
            className="MuiTypography-root MuiTypography-body1 css-1umm2mn-MuiTypography-root"
          >
            2019 All rights reserved.
          </p>
          <div
            className="MuiGrid-root MuiGrid-container MuiGrid-spacing-xs-1 MuiGrid-direction-xs-column css-k9gnbd-MuiGrid-root"
          >
            <div
              className="MuiGrid-root MuiGrid-item css-15zuw48-MuiGrid-root"
            >
              <a
                className="MuiBox-root css-1rp3oq8"
                href="https://facebook.com/"
              >
                <svg
                  aria-hidden={true}
                  className="MuiSvgIcon-root MuiSvgIcon-fontSizeMedium css-i4bv87-MuiSvgIcon-root"
                  data-testid="FacebookIcon"
                  focusable="false"
                  viewBox="0 0 24 24"
                >
                  <path
                    d="M5 3h14a2 2 0 0 1 2 2v14a2 2 0 0 1-2 2H5a2 2 0 0 1-2-2V5a2 2 0 0 1 2-2m13 2h-2.5A3.5 3.5 0 0 0 12 8.5V11h-2v3h2v7h3v-7h3v-3h-3V9a1 1 0 0 1 1-1h2V5z"
                  />
                </svg>
              </a>
              <a
                className="MuiBox-root css-1rp3oq8"
                href="https://twitter.com/"
              >
                <svg
                  aria-hidden={true}
                  className="MuiSvgIcon-root MuiSvgIcon-fontSizeMedium css-i4bv87-MuiSvgIcon-root"
                  data-testid="TwitterIcon"
                  focusable="false"
                  viewBox="0 0 24 24"
                >
                  <path
                    d="M22.46 6c-.77.35-1.6.58-2.46.69.88-.53 1.56-1.37 1.88-2.38-.83.5-1.75.85-2.72 1.05C18.37 4.5 17.26 4 16 4c-2.35 0-4.27 1.92-4.27 4.29 0 .34.04.67.11.98C8.28 9.09 5.11 7.38 3 4.79c-.37.63-.58 1.37-.58 2.15 0 1.49.75 2.81 1.91 3.56-.71 0-1.37-.2-1.95-.5v.03c0 2.08 1.48 3.82 3.44 4.21a4.22 4.22 0 0 1-1.93.07 4.28 4.28 0 0 0 4 2.98 8.521 8.521 0 0 1-5.33 1.84c-.34 0-.68-.02-1.02-.06C3.44 20.29 5.7 21 8.12 21 16 21 20.33 14.46 20.33 8.79c0-.19 0-.37-.01-.56.84-.6 1.56-1.36 2.14-2.23z"
                  />
                </svg>
              </a>
              <a
                className="MuiBox-root css-1rp3oq8"
                href="https://github.com/"
              >
                <svg
                  aria-hidden={true}
                  className="MuiSvgIcon-root MuiSvgIcon-fontSizeMedium css-i4bv87-MuiSvgIcon-root"
                  data-testid="GitHubIcon"
                  focusable="false"
                  viewBox="0 0 24 24"
                >
                  <path
                    d="M12 1.27a11 11 0 00-3.48 21.46c.55.09.73-.28.73-.55v-1.84c-3.03.64-3.67-1.46-3.67-1.46-.55-1.29-1.28-1.65-1.28-1.65-.92-.65.1-.65.1-.65 1.1 0 1.73 1.1 1.73 1.1.92 1.65 2.57 1.2 3.21.92a2 2 0 01.64-1.47c-2.47-.27-5.04-1.19-5.04-5.5 0-1.1.46-2.1 1.2-2.84a3.76 3.76 0 010-2.93s.91-.28 3.11 1.1c1.8-.49 3.7-.49 5.5 0 2.1-1.38 3.02-1.1 3.02-1.1a3.76 3.76 0 010 2.93c.83.74 1.2 1.74 1.2 2.94 0 4.21-2.57 5.13-5.04 5.4.45.37.82.92.82 2.02v3.03c0 .27.1.64.73.55A11 11 0 0012 1.27"
                  />
                </svg>
              </a>
              <a
                className="MuiBox-root css-1rp3oq8"
                href="https://facebook.com/"
              >
                <svg
                  aria-hidden={true}
                  className="MuiSvgIcon-root MuiSvgIcon-fontSizeMedium css-i4bv87-MuiSvgIcon-root"
                  data-testid="FacebookIcon"
                  focusable="false"
                  viewBox="0 0 24 24"
                >
                  <path
                    d="M5 3h14a2 2 0 0 1 2 2v14a2 2 0 0 1-2 2H5a2 2 0 0 1-2-2V5a2 2 0 0 1 2-2m13 2h-2.5A3.5 3.5 0 0 0 12 8.5V11h-2v3h2v7h3v-7h3v-3h-3V9a1 1 0 0 1 1-1h2V5z"
                  />
                </svg>
              </a>
              <a
                className="MuiBox-root css-1rp3oq8"
                href="https://github.com/"
              >
                <svg
                  aria-hidden={true}
                  className="MuiSvgIcon-root MuiSvgIcon-fontSizeMedium css-i4bv87-MuiSvgIcon-root"
                  data-testid="GitHubIcon"
                  focusable="false"
                  viewBox="0 0 24 24"
                >
                  <path
                    d="M12 1.27a11 11 0 00-3.48 21.46c.55.09.73-.28.73-.55v-1.84c-3.03.64-3.67-1.46-3.67-1.46-.55-1.29-1.28-1.65-1.28-1.65-.92-.65.1-.65.1-.65 1.1 0 1.73 1.1 1.73 1.1.92 1.65 2.57 1.2 3.21.92a2 2 0 01.64-1.47c-2.47-.27-5.04-1.19-5.04-5.5 0-1.1.46-2.1 1.2-2.84a3.76 3.76 0 010-2.93s.91-.28 3.11 1.1c1.8-.49 3.7-.49 5.5 0 2.1-1.38 3.02-1.1 3.02-1.1a3.76 3.76 0 010 2.93c.83.74 1.2 1.74 1.2 2.94 0 4.21-2.57 5.13-5.04 5.4.45.37.82.92.82 2.02v3.03c0 .27.1.64.73.55A11 11 0 0012 1.27"
                  />
                </svg>
              </a>
            </div>
          </div>
        </div>
        <div
          className="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-6 MuiGrid-grid-sm-4 MuiGrid-grid-md-2 css-8rpx47-MuiGrid-root"
        >
          <h5
            className="MuiTypography-root MuiTypography-h5 MuiTypography-gutterBottom css-10gsmdd-MuiTypography-root"
            marked="left"
          >
            Overview
          </h5>
          <ul
            className="MuiBox-root css-19brutj"
          >
            <li
              className="MuiBox-root css-mz9qt7"
            >
              <a
                className="MuiTypography-root MuiTypography-inherit MuiLink-root MuiLink-underlineHover css-1f7ol2x-MuiTypography-root-MuiLink-root"
                href="/"
                onBlur={[Function]}
                onFocus={[Function]}
              >
                Products
              </a>
            </li>
            <li
              className="MuiBox-root css-mz9qt7"
            >
              <a
                className="MuiTypography-root MuiTypography-inherit MuiLink-root MuiLink-underlineHover css-1f7ol2x-MuiTypography-root-MuiLink-root"
                href="/"
                onBlur={[Function]}
                onFocus={[Function]}
              >
                Use Cases
              </a>
            </li>
            <li
              className="MuiBox-root css-mz9qt7"
            >
              <a
                className="MuiTypography-root MuiTypography-inherit MuiLink-root MuiLink-underlineHover css-1f7ol2x-MuiTypography-root-MuiLink-root"
                href="/pricing"
                onBlur={[Function]}
                onFocus={[Function]}
              >
                Pricing
              </a>
            </li>
            <li
              className="MuiBox-root css-mz9qt7"
            >
              <a
                className="MuiTypography-root MuiTypography-inherit MuiLink-root MuiLink-underlineHover css-1f7ol2x-MuiTypography-root-MuiLink-root"
                href="/"
                onBlur={[Function]}
                onFocus={[Function]}
              >
                Blog
              </a>
            </li>
          </ul>
        </div>
        <div
          className="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-6 MuiGrid-grid-sm-4 MuiGrid-grid-md-2 css-8rpx47-MuiGrid-root"
        >
          <h5
            className="MuiTypography-root MuiTypography-h5 MuiTypography-gutterBottom css-10gsmdd-MuiTypography-root"
            marked="left"
          >
            Resources
          </h5>
          <ul
            className="MuiBox-root css-19brutj"
          >
            <li
              className="MuiBox-root css-mz9qt7"
            >
              <a
                className="MuiTypography-root MuiTypography-inherit MuiLink-root MuiLink-underlineHover css-1f7ol2x-MuiTypography-root-MuiLink-root"
                href="/"
                onBlur={[Function]}
                onFocus={[Function]}
              >
                Docs
              </a>
            </li>
            <li
              className="MuiBox-root css-mz9qt7"
            >
              <a
                className="MuiTypography-root MuiTypography-inherit MuiLink-root MuiLink-underlineHover css-1f7ol2x-MuiTypography-root-MuiLink-root"
                href="/"
                onBlur={[Function]}
                onFocus={[Function]}
              >
                Model Catalog
              </a>
            </li>
            <li
              className="MuiBox-root css-mz9qt7"
            >
              <a
                className="MuiTypography-root MuiTypography-inherit MuiLink-root MuiLink-underlineHover css-1f7ol2x-MuiTypography-root-MuiLink-root"
                href="/"
                onBlur={[Function]}
                onFocus={[Function]}
              >
                Dashboard
              </a>
            </li>
            <li
              className="MuiBox-root css-mz9qt7"
            >
              <a
                className="MuiTypography-root MuiTypography-inherit MuiLink-root MuiLink-underlineHover css-1f7ol2x-MuiTypography-root-MuiLink-root"
                href="/"
                onBlur={[Function]}
                onFocus={[Function]}
              >
                Terms of Use
              </a>
            </li>
          </ul>
        </div>
        <div
          className="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-6 MuiGrid-grid-sm-4 MuiGrid-grid-md-2 css-8rpx47-MuiGrid-root"
        >
          <h5
            className="MuiTypography-root MuiTypography-h5 MuiTypography-gutterBottom css-10gsmdd-MuiTypography-root"
            marked="left"
          >
            Company
          </h5>
          <ul
            className="MuiBox-root css-19brutj"
          >
            <li
              className="MuiBox-root css-mz9qt7"
            >
              <a
                className="MuiTypography-root MuiTypography-inherit MuiLink-root MuiLink-underlineHover css-1f7ol2x-MuiTypography-root-MuiLink-root"
                href="https://alwaysai.co/"
                onBlur={[Function]}
                onFocus={[Function]}
              >
                About
              </a>
            </li>
            <li
              className="MuiBox-root css-mz9qt7"
            >
              <a
                className="MuiTypography-root MuiTypography-inherit MuiLink-root MuiLink-underlineHover css-1f7ol2x-MuiTypography-root-MuiLink-root"
                href="/"
                onBlur={[Function]}
                onFocus={[Function]}
              >
                Careers
              </a>
            </li>
            <li
              className="MuiBox-root css-mz9qt7"
            >
              <a
                className="MuiTypography-root MuiTypography-inherit MuiLink-root MuiLink-underlineHover css-1f7ol2x-MuiTypography-root-MuiLink-root"
                href="/"
                onBlur={[Function]}
                onFocus={[Function]}
              >
                Press
              </a>
            </li>
            <li
              className="MuiBox-root css-mz9qt7"
            >
              <a
                className="MuiTypography-root MuiTypography-inherit MuiLink-root MuiLink-underlineHover css-1f7ol2x-MuiTypography-root-MuiLink-root"
                href="/"
                onBlur={[Function]}
                onFocus={[Function]}
              >
                Privacy
              </a>
            </li>
          </ul>
        </div>
        <div
          className="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-6 MuiGrid-grid-sm-4 MuiGrid-grid-md-2 css-8rpx47-MuiGrid-root"
        >
          <h5
            className="MuiTypography-root MuiTypography-h5 MuiTypography-gutterBottom css-10gsmdd-MuiTypography-root"
            marked="left"
          >
            Contact
          </h5>
          <ul
            className="MuiBox-root css-19brutj"
          >
            <li
              className="MuiBox-root css-mz9qt7"
            >
              <p
                className="MuiTypography-root MuiTypography-body1 css-cltehk-MuiTypography-root"
              >
                Support
              </p>
            </li>
            <li
              className="MuiBox-root css-mz9qt7"
            >
              <p
                className="MuiTypography-root MuiTypography-body1 css-cltehk-MuiTypography-root"
              >
                555-555-5555
              </p>
            </li>
            <li
              className="MuiBox-root css-mz9qt7"
            >
              <a
                className="MuiTypography-root MuiTypography-inherit MuiLink-root MuiLink-underlineHover css-1f7ol2x-MuiTypography-root-MuiLink-root"
                href="mailto:support@alwaysai.co"
                onBlur={[Function]}
                onFocus={[Function]}
              >
                support@alwaysai.co
              </a>
            </li>
            <li
              className="MuiBox-root css-mz9qt7"
            >
              <p
                className="MuiTypography-root MuiTypography-body1 css-cltehk-MuiTypography-root"
              >
                720 Stevens Ave #270
              </p>
            </li>
            <li
              className="MuiBox-root css-mz9qt7"
            >
              <p
                className="MuiTypography-root MuiTypography-body1 css-cltehk-MuiTypography-root"
              >
                Solana Beach, Ca 92075
              </p>
            </li>
          </ul>
        </div>
      </div>
    </div>
  </footer>
</div>
`;
